src/useWrappedRefWithWarning.js

```diff
@@ -0,0 +1,23 @@
+import invariant from 'invariant';
+import { useCallback } from 'react';
+import useMergedRefs from '@restart/hooks/useMergedRefs';
+
+const isReactClassComponent = c => c && c.isReactComponent;
```

Comment:
surely you can just inline this :p

Reply:
yes, i thought originally the sigil was on the constructor